epregressions/energyplus.py

```diff
@@ -171,8 +172,11 @@ def execute_energyplus(e_args: ExecutionArguments):
 
         # Execute EnergyPlus
         try:
+            command_line = energyplus
+            if os.path.exists('in.epJSON'):
+                command_line = energyplus + ' in.epJSON'
             std_out += subprocess.check_output(
-                energyplus, shell=True, stdin=subprocess.DEVNULL, stderr=subprocess.PIPE
+                command_line, shell=True, stdin=subprocess.DEVNULL, stderr=subprocess.PIPE
```

Comment:
By default, with no arguments, EnergyPlus will try to run `in.idf`.  It's a different story that we should be using the command line interface here, but for now, we need to at least tell it to use the `in.epJSON` file.